### PR TITLE
Use bigger queue size

### PR DESCRIPTION
**Pull request by style95**
Handle https://github.com/apache/openwhisk/issues/5452

## Description
When sending concurrent requests, a queue full error happens.
So I increased the queue size according to [this value in the nodejs runtime repo](https://github.com/apache/openwhisk-runtime-nodejs/blob/master/tests/src/test/scala/runtime/actionContainers/NodeJsConcurrentTests.scala#L32).

## Related issue and scope

- [x] I opened an issue to propose and discuss this change (#5452)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Scheduler
- [x] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md#coding-standards) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

File: common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/AkkaContainerClient.scala [hunk 1]
```diff
@@ -226,7 +216,7 @@ object AkkaContainerClient {
     tid: TransactionId,
     as: ActorSystem,
     ec: ExecutionContext): Seq[(Int, Option[JsObject])] = {
-    val connection = new AkkaContainerClient(host, port, timeout, 1)
+    val connection = new AkkaContainerClient(host, port, timeout, queueSize)
```
Comment (style95) [inline common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/AkkaContainerClient.scala:219]: I confirmed this change fix the CI tests of nodejs runtime.

**Comment by style95** [comment]
@joni-jones 
Could you take a look?

**Comment by joni-jones** [comment]
I see that the default value for `max-connections` is 128. So the queue might consume slightly more memory, but it should not have any impact. Overall, I think this fix makes sense.

**Review by dgrove-oss — APPROVED** [review]
lgtm.  Merging.

**Pull request merged**